wire-runtime/src/test/proto-java/com/squareup/wire/protos/person/Person.java

```diff
@@ -219,6 +220,22 @@ public static PhoneType fromValue(int value) {
     public int getValue() {
       return value;
     }
+
+    private static final class ProtoAdapter_PhoneType extends EnumAdapter<PhoneType> {
```

Comment:
That's fine, I just hate having to look at adapter code, but I guess that is the price we paid to get rid of reflection.